sklearn/tree/setup.py

```diff
@@ -31,7 +31,11 @@ def configuration(parent_package="", top_path=None):
                          extra_compile_args=["-O3"])
 
     config.add_subpackage("tests")
-    config.add_data_files('_criterion.pxd')
+    config.add_data_files("_tree.pxd")
```

Comment:
Alphabetical order?

Reply:
do you mean I should add pxd files in alphabetical order